lib/runner.js

```diff
@@ -99,7 +105,8 @@ Runner.prototype._addTestResult = function (result) {
 		error: result.reason,
 		type: test.metadata.type,
 		skip: test.metadata.skipped,
-		todo: test.metadata.todo
+		todo: test.metadata.todo,
+		exclusive: test.metadata.exclusive
```

Comment:
Why is this necessary? Presumably `.only` already works without needing this prop.